package.json

```diff
@@ -5,17 +5,33 @@
   "engines": {
     "node": ">=10"
   },
-  "dependencies": {},
   "devDependencies": {
     "@matterlabs/hardhat-zksync-deploy": "^0.6.3",
     "@matterlabs/hardhat-zksync-solc": "^0.3.16",
     "@matterlabs/hardhat-zksync-verify": "^0.1.5",
-    "@nomiclabs/hardhat-etherscan": "^3.1.7",
+    "@nomicfoundation/hardhat-toolbox": "^2.0.2",
+    "ethereum-waffle": "^3.4.4",
```

Comment:
Do we need `ethereum-waffle` here?

Reply:
Sure, we need it in https://github.com/vareger/zksync-uniswap-permit2/blob/added-hardhat-test/test/zksync-tests/shared/expect.ts, because test’s on zksync require special config for chai-mocha.